packages/fxa-auth-server/lib/customs.js

```diff
@@ -205,6 +209,31 @@ class CustomsClient {
       throw this.error.requestBlocked(unblock);
     }
   }
+
+  logHttpAgentStatus() {
+    if (this.axiosInstance && this.statsd) {
+      this.logStatus(this.httpAgent, 'httpAgent');
+      this.logStatus(this.httpsAgent, 'httpsAgent');
+    }
+  }
+
+  logStatus(agent, name) {
+    if (agent) {
+      const status = agent.getCurrentStatus();
+      this.statsd.gauge(`${name}.createSocketCount`, status.createSocketCount);
```

Comment:
Not sure if it is better to do a guage or increment statsd metric 🤔

Reply:
I don't know how gauge is gonna work when aggregated  across all the pods.  It would work on a per pod basis perhaps.